server/fleet/host_certificates_test.go

```diff
@@ -46,9 +52,14 @@ func TestExtractHostCertificateNameDetails(t *testing.T) {
 			expected: &expected,
 		},
 		{
-			name:  "invalid format with extra slash",
-			input: "/C=US/O=Fleet Device Management Inc./OU=Fleet Device Management Inc./CN=FleetDM/invalid",
-			err:   true,
```

Comment:
It is an edge case, but we still expect this will error if for whatever reason the extra slash wasn't escaped as "\/" by osquery, right?  I think that is ok and perhaps desirable as logs would alert us in the event that osquery (or an underlying library) randomly changed its approach to escape sequences. Let's add this test back as a way to assert the expected behavior for future reference.